src/Raven.Client/Documents/Session/Loaders/IncludeBuilder.cs

```diff
@@ -331,5 +375,50 @@ private void WithAlias(Expression<Func<T, string>> path)
             if (Alias == null)
                 Alias = path.Parameters[0].Name;
         }
+
+        private void IncludeTimeSeries(string path, string name, DateTime from, DateTime to)
+        {
+            if (string.IsNullOrWhiteSpace(name))
+                throw new ArgumentNullException(nameof(name));
+
+            if (TimeSeriesToIncludeBySourcePath == null)
+            {
+                TimeSeriesToIncludeBySourcePath = new Dictionary<string, HashSet<TimeSeriesRange>>();
+            }
+
+            if (TimeSeriesToIncludeBySourcePath.TryGetValue(path, out var hashSet) == false)
```

Comment:
`TimeSeriesToIncludeBySourcePath` should be `TimeSeriesToIncludeBySourceAlias` and the `string path`, etc should be changed to reflect that

Reply:
done